sycl/doc/extensions/experimental/sycl_ext_oneapi_bindless_images.asciidoc

```diff
@@ -428,6 +428,15 @@ The second way to allocate image memory is to use USM allocations. SYCL already
 provides a number of USM allocation functions. This proposal would add another,
 pitched memory allocation, through `pitched_alloc_device`.
 
+Bindless images can be backed by device, host, or shared USM memory allocations.
+
+[NOTE]
+====
+Bindless image memory from USM device allocations is generally supported for all
+backends, while host and shared USM allocations depend on the SYCL backend as well as
+the device capabilities.
```

Comment:
With recent internal testing, we can say that Host USM allocations are also generally supported on all devices. If they weren't, we would need an additional query.
Shared USM seems to be the odd one out, for which we already have a query.

```suggestion
Bindless image memory from USM device and host allocations is generally supported for all
backends, while shared USM allocations depend on the SYCL backend as well as
the device capabilities.
```

Reply:
Thanks! Yeah, I agree to generalise this so as to avoid the need for a device query in the case of host USM.